reference/performances_guide/index.rst

```diff
@@ -40,3 +40,57 @@ Here is the complete formula to check if you have, by far, more indexes than the
 .. warning::
 
     If your collection of products requires more than 64 indexes, please contact us (we've an Elastic Search implementation).
+
+More than 10k attributes?
+-------------------------
+
+The number of attributes has impacts on different screens of the PIM.
```

Comment:
I would rather say something like "The number of attributes will impact performances of the PIM in several ways".
What do you think ?